operations/deployment/terraform/postgres.tf

```diff
@@ -1,6 +1,6 @@
 resource "aws_security_group" "pg_security_group" {
-  count = var.enable_postgres == "true" ? 1 : 0
-  name        = var.security_group_name_pg
```

Comment:
If we introduce other providers in the future, it would be easier to manage them. We could rearrange this once we do commons. 
Checked GCP and most of the names are different. :disappointed: